web/src/components/Carousel.jsx

```diff
@@ -0,0 +1,65 @@
+import { h, Component } from 'preact';
+import Flickity from 'flickity';
+import 'flickity/css/flickity.css';
```

Comment:
Would really prefer to not start pulling in random packages for single-use items – especially those that require CSS like this.